macros/count_tokens.sql

```diff
@@ -0,0 +1,7 @@
+{% macro count_tokens(column_name) -%}
+  {{ return(adapter.dispatch('count_tokens', 'zendesk')(column_name)) }}
+{%- endmacro %}
+
+{% macro default__count_tokens(column_name) %}
+  {{ dbt.length(column_name) }} / 4
```

Comment:
Based on [this doc](https://help.openai.com/en/articles/4936856-what-are-tokens-and-how-to-count-them) and internal discussion, approximate count of tokens is appropriate.